src/marqo_commons/model_registry/model_properties_data/random_properties.py

```diff
@@ -18,29 +19,31 @@ class RandomModelProperties(ModelProperties):
     tokens: int
     notes: str = ""
 
+    @classmethod
+    def list_model_properties(cls) -> Dict[str, T]:
+        return  {
+            "random": RandomModelProperties(
+                name="random",
+                dimensions=384,
+                tokens=128,
+            ),
+            "random/large": RandomModelProperties(
+                name="random/large",
+                dimensions=768,
+                tokens=128,
+            ),
+            "random/small": RandomModelProperties(
+                name="random/small",
+                dimensions=32,
+                tokens=128,
+            ),
+            "random/medium": RandomModelProperties(
+                name="random/medium",
+                dimensions=128,
+                tokens=128,
+            ),
+        }
+
 @convert_model_properties_to_dict
 def _get_random_properties() -> Dict:
-    RANDOM_MODEL_PROPERTIES = {
-        "random": RandomModelProperties(
-            name="random",
-            dimensions=384,
-            tokens=128,
-        ),
-        "random/large": RandomModelProperties(
-            name="random/large",
-            dimensions=768,
-            tokens=128,
-        ),
-        "random/small": RandomModelProperties(
-            name="random/small",
-            dimensions=32,
-            tokens=128,
-        ),
-        "random/medium": RandomModelProperties(
-            name="random/medium",
-            dimensions=128,
-            tokens=128,
-        ),
-    }
-
-    return RANDOM_MODEL_PROPERTIES
+    return RandomModelProperties.list_model_properties()
```

Comment:
It may be good to have a docstring on each of these old methods as well explaining that it model properties are converted to dict.

Reply:
actually these old functions are internal functions so I'd say a docstring is pointless here, I could add a docstring to the calling function i.e. `model_registry.get_model_properties_json()` but I feel like the name already implies what it does, so a docstring won't really add much value there